runtime/BUILD.bazel

```diff
@@ -28,6 +28,7 @@ go_library(
         "@com_github_golang_protobuf//descriptor:go_default_library_gen",
         "@com_github_golang_protobuf//jsonpb:go_default_library_gen",
         "@com_github_golang_protobuf//proto:go_default_library",
+        "@com_github_golang_protobuf//ptypes:go_default_library_gen",
```

Comment:
This is suspect. What does it mean that it's `go_default_library_gen`? How do I know which version of `protoc-gen-go` is used to generate these files?

Reply:
The plot thickens: https://github.com/bazelbuild/rules_go/blob/master/third_party/com_github_golang_protobuf-extras.patch. This seems to be hardcoded. Maybe we need to override it?

Reply:
This might've been fixed upstream, but I can't tell: https://github.com/bazelbuild/rules_go/commit/2634df77091483c4e63b8523d5d3815bfbf3f2eb. Probably worth just waiting for the next `rules_go` release and see.

Reply:
Worked around this issue by avoiding `ptypes`.